Teradata/covalent
# chore(): upgrade ng-packagr from 2.0.0-rc.10 to 2.0.0 stable

## Description
Upgraded ng-packagr

### What's included?

- Upgraded ng-packagr from 2.0.0-rc.10 to stable 2.0.0 

#### Test Steps (Covalent build/test)
- Verify you are using node v8.9.x
- `npm run reinstall`
- To make the next steps a little easier go to `scripts/build-release` and add this code to the bottom of the file to `npm pack` instead of doing it manually:
```
cd deploy/platform/core
npm pack
cd ../dynamic-forms
npm pack
cd ../highlight
npm pack
cd ../http
npm pack
cd ../markdown
npm pack
```
- Verify `npm run serve` works accordingly. Test all components [http://localhost:4200/](http://localhost:4200/)
- run `npm run build:release`
#### Test Steps (Product build)
- Clone AppCenter repo into the directory above this one.
- `yarn cache clean`
- `yarn install`
- `yarn remove @covalent/core`
- `yarn remove @covalent/highlight`
- `yarn remove @covalent/http`
- `yarn remove @covalent/markdown`
- `yarn add <file-path-to-*.tgz>` repeat for (core, highlight, http, markdown respectively) (EX: `yarn add ../covalent/deploy/platform/core/covalent-core-1.0.0-rc.4.tgz`)
- `npm rebuild node-sass`
- `yarn upgrade @angular/material`
- `yarn upgrade @angular/cdk`
- Verify you are using node v8.9.x. 
- Verify `npm run serve` works correctly
- Verify `./node_modules/@angular/cli/bin/ng build --aot --prod --sourcemap=false --build-optimizer` successfully runs
- Verify `./node_modules/@angular/cli/bin/ng serve --aot --prod --sourcemap=false --build-optimizer --proxy-config proxy.conf.json` successfully runs and you can access the local site [http://localhost:4200/](http://localhost:4200/)
- Modify the import paths (primary/secondary entry point usage) in app.module and trying the prod build again it should run successfully.

#### General Tests for Every PR

- [ ] `npm run build:release` still works.

closes https://github.com/Teradata/covalent/issues/1099
